app/upw/controllers/taskList.utils.js

```diff
@@ -109,7 +109,9 @@ const getTaskList = (baseUrl = '', steps = {}, answers = {}, riskFlags = []) =>
     },
   ]
 
-  if (answers.gender_identity && answers.gender_identity[0] === 'MALE') {
+  const genderIdentity = answers.gender_identity || []
+
+  if (!genderIdentity.length, genderIdentity.includes('MALE')) {
```

Comment:
Won't this fire if gender is FEMALE too?

Reply:
It shouldn't, the answer should be an array here so we're asserting on the string?

Reply:
So if the user selected 'FEMALE' we'd still show the section (I may be grasping the wrong end of the stick here)?

Reply:
We should hide the section if it's not relevant, in this case in this if the answer (array) is empty or includes `male`, otherwise we'll show the placement preference section